optimize/self-managed/optimize-deployment/migration-update/camunda-8/3.13_8.5-to-8.6.md

```diff
@@ -23,12 +23,6 @@ Here you will find information about:
 
 Optimize no longer gathers telemetry data, and this is removed from the UI and Elasticsearch. The associated key in the configuration file (`telemetry.telemetryEndpoint`) was removed.
 
-## Changes in supported environments
-
-### Camunda 7
-
-Optimize now requires at least Camunda 7 `7.20.0` and supports up to `7.22.0+`. Camunda 7 `7.19.x` is no longer supported.
-
 ### Java
 
 Optimize now supports Java 21+.
```

Comment:
In the Docker tag section, do you think we should be firmer about warning C8 users against using the `latest` tag? We could even have a warning box saying not to use it?

Reply:
I did that in https://github.com/camunda/camunda-docs/pull/4220
Let me know if a warning is still need and i can update it there.